compiler/src/compile.re

```diff
@@ -41,8 +41,7 @@ type error =
 
 exception InlineFlagsError(Location.t, error);
 
-let default_output_filename = name =>
-  Filepath.String.remove_extension(name) ++ ".gr.wasm";
+let default_output_filename = name => name ++ ".wasm";
```

Comment:
We'll need to call this out in a changelog entry, since we'll consider this breaking.

Reply:
This actually isn't breaking, it's just that the input to this function now contains the `.gr` extension; the output filepaths will be the same for `.gr` files

Reply:
I gotcha.